go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/fatih/color v1.13.0
 	github.com/ferranbt/fastssz v0.0.0-20220103083642-bc5fefefa28b
 	github.com/gdamore/tcell/v2 v2.4.1-0.20210905002822-f057f0a857a1
-	github.com/glendc/go-external-ip v0.0.0-20200601212049-c872357d968e
+	github.com/glendc/go-external-ip v0.1.0
```

Comment:
should explicitly mention the dependency upgrade change in the commits + PR description

Reply:
Concur, I usually don't upgrade libraries without a good reason to avoid supply chain attacks. I'll look through this to see if it provides anything new or introduces anything scary.

Reply:
In this case, it looks like the protocol specification flag was introduced in v0.1.0, so this update is required. The diff doesn't show anything alarming: https://github.com/glendc/go-external-ip/compare/c872357d968e..v0.1.0